CONTRIBUTING.md

```diff
@@ -1,106 +1,186 @@
+# Contributing
+
+| Component            | Build Status                                                                                                                                                                                                        |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Management Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/net/net%20-%20mgmt%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=529&branchName=master) |
+| Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master)                           |
+
+# Prerequisites:
+
+Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+
+**Client Libraries** are sdks used to interact with azure resources at application run time while **Management Libraries** are those used to manage (create/modify/delete) Azure resources.
+<br/><br/>
+
+# Client Libraries
+
+## TO BUILD:
+
+### Single Service from Command Line
+
+1. Open VS 2019 Command Prompt
+2. Navigate to service directory e.g. _"sdk\eventhub"_
+3. Invoke `dotnet build`
+4. or Build the **service.proj** in the repo root, passing the directory name of the specific service as a property. e.g. `dotnet build eng\service.proj /p:ServiceDirectory=eventhub`
+
+### Single Service from Visual Studio
+
+1. Open any data-plane solution e.g. _"sdk\eventhub\Microsoft.Azure.EventHubs.sln"_
+2. Build solution from Visual Studio
+
+### All Client Services from Command Line
+
+1. Open VS 2019 Command Prompt
+2. Navigate to repository root directory
+3. Invoke `dotnet build eng\service.proj`
+
+## TO TEST:
+
+### Single Service from Command Line
+
+1. Open VS 2019 Command Prompt
+2. Navigate to service directory e.g. _"sdk\eventhub"_
+3. Invoke `dotnet test --filter TestCategory!=Live` _(Skips live tests)_
+4. or run test against **service.proj** in the repo root, passing the directory name of the specific service as a property. e.g. `dotnet test eng\service.proj /p:ServiceDirectory=eventhub --filter TestCategory!=Live`
+
+### Single Service from Visual Studio
+
+1. Build.
+2. Test Explorer window will get populated with tests. Select test and `Run` or `Debug`
+
+### All Client Services from Command Line
+
+1. Open VS 2019 Command Propmpt
+2. Navigate to repository root directory
+3. Invoke `dotnet test eng\service.proj --filter TestCategory!=Live`
+   <br/><br/>
+
+# Management Libraries
+
+## TO BUILD:
+
+### If you are building from Visual Studio, add a nuget feed source that points to < root >\tools\LocalNugetFeed directory
```

Comment:
You don't have to mention this explicitly, because that nuget feed get's added automatically in VS due nuget.config